src/PowerShellEditorServices/Console/ConsoleReadLine.cs

```diff
@@ -19,8 +19,9 @@ namespace Microsoft.PowerShell.EditorServices.Console
 
     internal class ConsoleReadLine
     {
-        #region Private Fields
+        #region Private Field
 
+        private bool isUnix = false;
```

Comment:
Yeah, not sure why I initialized it.  I was trying to scratch out some code in VS Code on Mac OSX and I think the lack of IntelliSense was screwing up my senses.  Something about the net451 TargetFramework was making OmniSharp fail to load the projects correctly.